source/release-notes/4-0-0.rst

```diff
@@ -64,7 +70,19 @@ New Features
 Updates
 ^^^^^^^
 
-* Upgraded Elasticsearch version to 7.16.2
+* Unlock permission added and can be assigned to a role |br|
+  See `here <https://github.com/craftercms/craftercms/issues/3690>`__ for more information
+
+* Added support for deployment mode for deployer processors |br|
+  Supported modes currently are ``PUBLISH`` and ``SEARCH_INDEX``.
+  See :ref:`here <crafter-deployer-api-target-deploy>` and :ref:`here <crafter-deployer-api-target-deploy>`
+  for more information
+
+* Upgraded Tika to 2.3.0
+
+* Upgraded SpringBoot to 2.6.4
```

Comment:
Remove

Reply:
Also indicate that Tomcat is v9 (but remove the minor and patch numbers.

Also add that Java is upgraded to v11